tests/integration/e2e/helpers.py

```diff
@@ -120,3 +124,95 @@ async def fetch_action_stop_process(unit: Unit) -> Dict:
 def get_random_topic() -> str:
     """Return a random topic name."""
     return f"topic-{''.join(random.choices(string.ascii_lowercase, k=4))}"
+
+
+def _get_exec_args_params() -> ExecArgs:
+    if SUBSTRATE == "k8s":
+        container_arg = "--container kafka"
+        sudo_arg = ""
+        bin_cmd = "/opt/kafka/bin/kafka-{sub}.sh"
+        config_file = "/etc/kafka/client.properties"
+    else:
+        container_arg = ""
+        sudo_arg = "sudo -i"
+        bin_cmd = "charmed-kafka.{sub}"
+        config_file = "/var/snap/charmed-kafka/current/etc/kafka/client.properties"
+
+    return ExecArgs(container_arg, sudo_arg, bin_cmd, config_file)
+
+
+def create_topic(model_full_name: str, app_name: str, topic: str) -> None:
+    """Helper to create a topic.
+
+    Args:
+        model_full_name: Juju model
+        app_name: Kafka app name in the Juju model
+        topic: the desired topic to configure
+    """
+    args = _get_exec_args_params()
+    try:
+        check_output(
+            f"JUJU_MODEL={model_full_name} juju ssh {args.container_arg} {app_name}/leader {args.sudo_arg} "
+            f"'{args.bin_cmd.format(sub='topics')} --create --topic {topic} --bootstrap-server localhost:19092 "
+            f"--command-config {args.config_file}'",
+            stderr=STDOUT,
+            shell=True,
+            universal_newlines=True,
+        )
+
+    except CalledProcessError as e:
+        logger.error(f"command '{e.cmd}' return with error (code {e.returncode}): {e.output}")
+        raise
+
+
+def write_topic_message_size_config(
```

Comment:
There is no good reason per se, it is the result of the iterative work on this ticket.
At first, there was no `create_topic`, I would relate to a producer and use the relation topic. However, since we have multiple test files, I thought it would be better to enforce separation of concerns and keep the "surface attack" as small as possible